### DISCUSSION TITLE
No License for Nuget Package Avalonia.Angle.Windows.Natives

### DISCUSSION THREAD

**Discussion by BenAigan** [AvaloniaUI/Avalonia#6122] [Q&A]
Can you please get a license applied to https://www.nuget.org/packages/Avalonia.Angle.Windows.Natives/2.1.0.2020091801 as my enterprise license scanning software rejects the use of it.

**Comment by kekekeks** [comment]
The license text is [here](https://github.com/AvaloniaUI/angle/blob/master/LICENSE). I've added it to the package's readme, but nuget won't allow metadata changes without re-uploading a new version of the package.
